pkgs/development/python-modules/grpc-stubs/default.nix

```
@@ -0,0 +1,60 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  grpcio,
+  pythonOlder,
+  pytest,
+  pytestCheckHook,
+  pytest-mypy-plugins,
+  types-protobuf,
+  cffi,
+  filelock,
+  hatchling,
+  poetry-core,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "grpc-stubs";
+  version = "1.53.0.5";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "shabbyrobe";
+    repo = "${pname}";
```

We want to avoid linking repo name with pname:
- #277994
```suggestion
    repo = "grpc-stubs";
```